Yank/tests/test_restraints.py

```diff
@@ -221,24 +221,40 @@ def test_partial_parametrization():
         ('Harmonic', dict(spring_constant=2.0*unit.kilojoule_per_mole/unit.nanometer**2,
                           restrained_receptor_atoms=[5])),
         ('FlatBottom', dict(well_radius=1.0*unit.angstrom, restrained_ligand_atoms=[130])),
-        ('Boresch', dict(restrained_atoms=[106, 123, 117, 130, 131, 136],
+        ('Boresch', dict(restrained_ligand_atoms=[130, 131, 136],
```

Comment:
Doesn't this test also need the "restrainted_receptor_atoms"? And if you are wanting to just randomly choose the receptor atoms as per how it is coded right now, it will also randomly rechoose the ligand atoms.